packages/python-sdk/e2b/sandbox/process.py

```diff
@@ -326,7 +327,7 @@ def exit_handler():
                 unsub_all()
             if on_exit:
                 try:
-                    on_exit(output.exit_code or 0)
+                    on_exit(*[output.exit_code or 0])
```

Comment:
why like this?

Reply:
I have type error otherwise because the function is accepting none or one argument.

Reply:
`output.exit_code or 0` is exactly one argument

Reply:
Slight correction:

> I have type error otherwise because the *function `on_exit` could be a function accepting none or a function accepting one argument*.